src/reproschema_to_fhir/fhir.py

```diff
@@ -210,94 +355,29 @@ def convert_to_fhir(self, reproschema_content: dict):
 
         if f"preamble" in reproschema_schema.keys():
             if isinstance(reproschema_schema[f"preamble"], dict):
-                group[f"text"] = reproschema_schema[f"preamble"]["en"]
+                group[f"text"] = reproschema_schema[f"preamble"][
+                    self.config.get_language()]
             elif isinstance(reproschema_schema[f"preamble"], str):
                 group[f"text"] = reproschema_schema[f"preamble"]
         else:
-            group[f"text"] = ""
+            group[f"text"] = " "
 
         group[f"type"] = f"group"
 
-        # create a pointer to the reproschema_items jsons
-        reproschema_items = OrderedDict(
-            [
-                (i, reproschema_content[i])
-                for i in reproschema_content.keys()
-                if i.startswith("items/")
-            ]
-        )
-
-        items = []
-        for item_path, item_json in reproschema_items.items():
-            curr_item = dict()
-
-            var_name = item_json["@id"]
-            curr_item[f"linkId"] = var_name
-
-            item_type = f"string"
-            if f"inputType" in item_json[f"ui"]:
-                if item_json[f"ui"][f"inputType"] == f"radio":
-                    item_type = f"choice"
-                elif item_json[f"ui"][f"inputType"] in (f"number", f"xsd:int"):
-                    item_type = f"integer"
-                # TODO: else what?
-
-            curr_item[f"type"] = item_type
-
-            # TODO: turn into a get_i18n function, which automatically
-            # uses a self.language arg to pull the relevant language from the dict
-            if f"question" in item_json and isinstance(
-                item_json[f"question"], dict
-            ):
-                curr_item[f"text"] = str(item_json[f"question"][f"en"])
-            else:
-                curr_item[f"text"] = str(item_json[f"prefLabel"])
+        # create a pointer to the reproschema_items jsons and match the question
+        reproschema_items = OrderedDict([(i, reproschema_content[i])
+                                         for i in reproschema_content.keys()
```

Comment:
you can use `i, value in reproschema_content.items()` instead of doing keys and indexing the dict again